tools/releases/changelog/github.go

```diff
@@ -202,6 +203,9 @@ type CommitInfo struct {
 	MaxDependency string
 }
 
+// titles look like: chore(deps): bump github.com/lib/pq from 1.10.6 to 1.10.7
+var dependabotPRTitleRegExp = regexp.MustCompile(`(chore\(deps\): bump [^ ]+) from ([^ ]+) to ([^ ]+)`)
```

Comment:
Nice, I think the only thing that won't be aggregated is kuma-gui, because there title has no from

Reply:
Kuma-gui adds a `> Changelog:` marker so it rolls up well see: https://github.com/kumahq/kuma/pull/5307 

For example here's the output atm:

```
* chore(deps): use latest kumahq/kuma-gui [#5265](https://github.com/kumahq/kuma/pull/5265) [#5272](https://github.com/kumahq/kuma/pull/5272) [#5281](https://github.com/kumahq/kuma/pull/5281) [#5307](https://github.com/kumahq/kuma/pull/5307) [#5321](https://github.com/kumahq/kuma/pull/5321) [#5332](https://github.com/kumahq/kuma/pull/5332) [#5346](https://github.com/kumahq/kuma/pull/5346) @kumahq/kuma-docs 
```